tests/unittest.jl

```diff
@@ -0,0 +1,78 @@
+include("../Tmp/data/ExampleMC.jl")
+include("../Tmp/data/ExampleWithVectorMember.jl")
+include("../Tmp/data/ExampleForCyclicDependency1.jl")
+include("../Tmp/data/ExampleForCyclicDependency2.jl")
+include("../Tmp/data/ExampleWithOneRelation.jl")
+include("../Tmp/data/ExampleCluster.jl")
+
+using Test
+@testset "All" begin
```

Comment:
```suggestion
@testset "Julia Bindings" begin
```
Just a slightly less generic way